docs/release-notes/1.2.0/release-1.2.0.md

```diff
@@ -0,0 +1,70 @@
+# [ML.NET](http://dot.net/ml) 1.2.0 
+## **General Availability**
+- **Microsoft.ML.TimeSeries**  
+    - Anomaly detection algorithms (Spike and Change Point):
+      - Independent and identically distributed.
+      - Singular spectrum analysis.
+      - Spectral residual from Azure Anomaly Detector/Kensho team.
+    - Forecasting models:
+      - Singular spectrum analysis.
+    - Prediction Engine for online learning
+      - Allows updating time series model with new observations at scoring so that the user does not have re-train the time series with old data each time.
+
+     [Samples](https://github.com/dotnet/machinelearning/tree/master/docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/TimeSeries)
+
+- **Microsoft.ML.OnnxTransformer**  
+   Allows scoring of ONNX models in the learning pipeline. Uses ONNX Runtime v0.4.
+      
+   [Sample](https://github.com/dotnet/machinelearning/blob/master/docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/ApplyOnnxModel.cs)
+
+- **Microsoft.ML.TensforFlow**  
+   Allows scoring of TensorFlow models in the learning pipeline. Uses TensforFlow v1.13. Very useful for image and text classification.
+
+   [Samples](https://github.com/dotnet/machinelearning/tree/master/docs/samples/Microsoft.ML.Samples/Dynamic/TensorFlow)
+
+## **New Features**
+- **Tree-based featurization** ([#3812](https://github.com/dotnet/machinelearning/pull/3812))
```

Comment:
Can we provide reference or more technical details?

Reply:
Here are two references.
- p. 9 at https://www.csie.ntu.edu.tw/~r01922136/kaggle-2014-criteo.pdf
- Section 3 at http://www.quinonero.net/Publications/predicting-clicks-facebook.pdf.
ML.NET's tree featurization trains a tree-based model and then maps input feature vector to several non-linear feature vectors. Those generated feature vectors are
- the leaves it falls into. It's a binary vector with ones happens at the indexes of reached leaves,
- the paths that the input vector passes before hitting the leaves, and
- the reached leaves values.